pkg/packaging/packaging.go

```diff
@@ -784,6 +838,38 @@ func pkgbuild(packageRoot, scriptsRoot, identifier, version, macPackageSigningKe
 	return nil
 }
 
+// distributionTemplateOptions is a struct which contains dynamic distribution
+// file parameters that will be rendered into a template.
+type distributionTemplateOptions struct {
+	PackageVersion    string
+	PackageFileName   string
+}
+
+// renderDistributionFile renders a distribution file to add to launcher's template.
+func renderDistributionFile(w io.Writer, options *distributionTemplateOptions) error {
+	distributionTemplate :=
+		`<?xml version="1.0" encoding="utf-8"?>
+<installer-gui-script minSpecVersion="1">
+    <pkg-ref id="com.launcher.launcher"/>
```

Comment:
This should be parameterized with the `identifier` option so that it's `com.{{.Identifier}}.launcher`